betterdgg/modules/passivestalk.js

```diff
@@ -8,21 +8,38 @@
         function PushChat(string) {
             destiny.chat.gui.push(new ChatInfoMessage(string));
         }
+        function updatePs(r){
+            var psList = r.join(',');
+            bdgg.settings.put('bdgg_passive_stalk', psList);
+            PushChat("Your passivestalk list has been updated");
+            document.querySelector('#bdgg_passive_stalk').value = bdgg.settings.get('bdgg_passive_stalk');
```

Comment:
Oh yeah, for sure. I saw that further down I did the same thing again. I tried to leave as much of the code untouched as possible but should probably just have rewritten it. Will do so now.